packages/block-editor/src/components/block-pattern-setup/style.scss

```diff
@@ -0,0 +1,124 @@
+.block-editor-block-pattern-setup {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: flex-start;
+	width: 100%;
+	border-radius: $radius-block-ui;
+	box-shadow: inset 0 0 0 $border-width $gray-900;
+	outline: 1px solid transparent; // Shown for Windows 10 High Contrast mode.
+
+	// TODO change to check parent.
+	&.view-mode-grid {
+		.block-editor-block-pattern-setup__toolbar {
+			justify-content: center;
+		}
+
+		.block-editor-block-pattern-setup__container {
+			display: grid;
+			grid-template-columns: 1fr 1fr;
+			grid-gap: $grid-unit-20;
+			padding: $grid-unit-20;
+			max-height: 550px;
+			overflow: auto;
+			margin: 0 $border-width $border-width $border-width;
+			width: calc(100% - #{ $border-width * 2 });
+			background: $white;
+
+			.block-editor-block-preview__container,
+			div[role="button"] {
+				cursor: pointer;
+			}
+
+			.block-editor-block-pattern-setup-list__item-title {
+				padding: $grid-unit-05;
+				font-size: $helptext-font-size;
+				text-align: center;
+			}
+
+			.block-editor-block-preview__container {
+				border-radius: $radius-block-ui;
+			}
+		}
+	}
+
+	.block-editor-block-pattern-setup__toolbar {
+		box-sizing: border-box;
+		position: relative;
+		padding: $grid-unit-20;
+		width: 100%;
+		text-align: left;
+		margin: 0;
+		color: $gray-900;
+		// Block UI appearance.
+		border-radius: $radius-block-ui $radius-block-ui 0 0;
+		background-color: $white;
+		box-shadow: inset 0 0 0 $border-width $gray-900;
+		outline: 1px solid transparent; // Shown for Windows 10 High Contrast mode.
+
+		display: flex;
+		flex-direction: row;
+		align-items: center;
+		justify-content: space-between;
+
+		.block-editor-block-pattern-setup__display-controls {
+			display: flex;
+		}
+
+		.block-editor-block-pattern-setup__navigation,
+		.block-editor-block-pattern-setup__actions {
+			width: calc(50% - 36px); // This ensures the grid / list toggle is centrally aligned.
```

Comment:
I wish there was a better way to center besides the 'magic' px values.  But I don't have any other suggestions.

Reply:
Me too, but I don't have any other suggestion either.. 😄